data/discovery/field/legislation.yaml

```diff
@@ -0,0 +1,6 @@
+cardinality: '1'
+datatype: text
```

Comment:
According to our desired proposal this should be a URL to legislation.gov.uk right?